app/models/document.rb

```diff
@@ -64,7 +64,7 @@ def tag_metadata
 
   def tag_labels_for(key)
     Array(attrs.fetch(key, []))
-      .map { |tag| tag.fetch("label") }
+      .map(&method(:get_metadata_label))
```

Comment:
To elaborate a little:

`Symbol#to_proc` returns a proc that sends the symbol as a message to the object given to the proc. `&:foo` is equivalent to the block `{ |x| x.foo }`.

`Method#to_proc` returns a proc that calls the method with the arguments given to the proc. `&method(:foo)` is equivalent to the block `{ |x| foo(x) }` (although it supports multiple arguments as well).